### PR TITLE
Dell.CommandUpdate.Universal version 4.5.0

### PR DESCRIPTION
**Pull request by rvtdadmin**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/66861)

### CODE REVIEW

File: manifests/d/Dell/CommandUpdateUniversal/4.5.0/Dell.CommandUpdateUniversal.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Dell.CommandUpdateUniversal
```
Comment (Trenly) [inline manifests/d/Dell/CommandUpdateUniversal/4.5.0/Dell.CommandUpdateUniversal.installer.yaml:4]:
```suggestion
PackageIdentifier: Dell.CommandUpdate.Universal
```

File: manifests/d/Dell/CommandUpdateUniversal/4.5.0/Dell.CommandUpdateUniversal.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Dell.CommandUpdateUniversal
```
Comment (Trenly) [inline manifests/d/Dell/CommandUpdateUniversal/4.5.0/Dell.CommandUpdateUniversal.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Dell.CommandUpdate.Universal
```

File: manifests/d/Dell/CommandUpdateUniversal/4.5.0/Dell.CommandUpdateUniversal.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Dell.CommandUpdateUniversal
```
Comment (Trenly) [inline manifests/d/Dell/CommandUpdateUniversal/4.5.0/Dell.CommandUpdateUniversal.yaml:4]:
```suggestion
PackageIdentifier: Dell.CommandUpdate.Universal
```

**Review by Trenly — CHANGES_REQUESTED** [review]
Because `Dell.CommandUpdate` already exists, this should be made as a sub-package. Remember when changing the Package Identifier to change the folder the manifest is in to match

**Comment by rvtdadmin** [comment]
When testing the manifest locally I'm getting a failure and it doesn't really seem to be with the manifest. 
Package installs but failure exit code is received.

Starting package install...
Installer failed with exit code: 2

LOG:
2022-07-19 16:41:36.824 [CORE] WinGet, version [1.3.1611-preview], activity [{54982B29-4D4A-44D5-A47B-66053361A1CD}]
2022-07-19 16:41:36.824 [CORE] OS: Windows.Desktop v10.0.19044.1826
2022-07-19 16:41:36.824 [CORE] Command line Args: winget  install --manifest C:\Users\User\manifests\d\Dell\CommandUpdate\Universal\4.5.0
2022-07-19 16:41:36.824 [CORE] Package: Microsoft.DesktopAppInstaller v1.18.1611.0
2022-07-19 16:41:36.824 [CORE] IsCOMCall:0; Caller: winget-cli
2022-07-19 16:41:36.851 [CLI ] WinGet invoked with arguments: 'install' '--manifest' 'C:\Users\User\manifests\d\Dell\CommandUpdate\Universal\4.5.0'
2022-07-19 16:41:36.854 [CLI ] Found subcommand: install
2022-07-19 16:41:36.854 [CLI ] Leaf command to execute: root:install
2022-07-19 16:41:36.863 [CLI ] Executing command: install
2022-07-19 16:41:36.886 [CLI ] Manifest fields: Name [Command | Update Windows Universal Application, 4.5.0, A00], Version [4.5.0]
2022-07-19 16:41:36.888 [CLI ] Starting installer selection.
2022-07-19 16:41:36.888 [CLI ] Completed installer selection.
2022-07-19 16:41:36.908 [CLI ] Found existing installer file at 'C:\Users\User\AppData\Local\Temp\WinGet\Dell.CommandUpdate.Universal.4.5.0\Dell-Command-Update-Windows-Universal-Application_601KT_WIN_4.5.0_A00_01.exe'. Verifying file hash.
2022-07-19 16:41:37.193 [CLI ] Existing installer file hash matches. Will use existing installer.
2022-07-19 16:41:37.193 [CLI ] Installer hash verified
2022-07-19 16:41:37.193 [CORE] Started applying motw using IAttachmentExecute to C:\Users\User\AppData\Local\Temp\WinGet\Dell.CommandUpdate.Universal.4.5.0\Dell-Command-Update-Windows-Universal-Application_601KT_WIN_4.5.0_A00_01.exe
2022-07-19 16:41:37.618 [CORE] Finished applying motw using IAttachmentExecute. Result: 0 IAttachmentExecute::Save() result: 0
2022-07-19 16:41:37.618 [REPO] Creating PredefinedInstalledSource with filter [ARP]
2022-07-19 16:41:37.618 [REPO] Creating new SQLite Index [4294967295.4294967295] at ':memory:'
2022-07-19 16:41:37.618 [SQL ] Opening SQLite connection: ':memory:' [6, 0]
2022-07-19 16:41:37.670 [REPO] Examining ARP entries for Machine | X64
2022-07-19 16:41:37.689 [REPO] Examining ARP entries for Machine | X86
2022-07-19 16:41:37.725 [REPO] Examining ARP entries for User | X64
2022-07-19 16:41:37.770 [CLI ] Installer args: /S
2022-07-19 16:41:37.771 [CLI ] Starting: 'C:\Users\User\AppData\Local\Temp\WinGet\Dell.CommandUpdate.Universal.4.5.0\Dell-Command-Update-Windows-Universal-Application_601KT_WIN_4.5.0_A00_01.exe' with arguments '/S'
2022-07-19 16:42:09.687 [CLI ] ShellExecute installer failed: 2
2022-07-19 16:42:09.687 [CLI ] Terminating context: 0x8a150006 at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\InstallFlow.cpp:193



WINGET INFO:

C:\WINDOWS\system32>winget --info
Windows Package Manager (Preview) v1.3.1611-preview
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19044.1826
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.18.1611.0

Logs: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir

Links
---------------------------------------------------------------------------
Privacy Statement   https://aka.ms/winget-privacy
License Agreement   https://aka.ms/winget-license
Third Party Notices https://aka.ms/winget-3rdPartyNotice
Homepage            https://aka.ms/winget
Windows Store Terms https://www.microsoft.com/en-us/storedocs/terms-of-sale

**Comment by JohnMcPMS** [comment]
See the [validation details](https://github.com/microsoft/winget-pkgs/pull/66861/checks?check_run_id=7419724195) for information on why the manifest isn't accepted:
```
Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\d\Dell\CommandUpdateUniversal\4.5.0', expected 'manifests\d\Dell\CommandUpdate\Universal\4.5.0'
```
If the installer returns anything but 0, we fail by default.  2 might be ok, you would have to check with their documentation.  If it is, you can add it to the success return codes.

File: manifests/d/Dell/CommandUpdateUniversal/4.5.0/Dell.CommandUpdateUniversal.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Dell.CommandUpdate.Universal
+PackageVersion: 4.5.0
```
Comment (ItzLevvie) [inline manifests/d/Dell/CommandUpdateUniversal/4.5.0/Dell.CommandUpdateUniversal.installer.yaml:5]:
```suggestion
PackageVersion: 4.5.0
InstallerSuccessCodes:
- 2
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by rvtdadmin** [comment]
I do not know how to change the manifest file path to match the PackageIdentifier now that the PackageIdentifier has been updated as suggested.

Dell.CommandUpdate.Universal

Thank you

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**